server/services/notification.service.js

```diff
@@ -31,7 +25,7 @@ export const markNotificationAsRead = async ({ userId, notificationId }) => {
   try {
     return Notification.findOneAndUpdate(
       { _id: notificationId, userId },
-      { $set: { status: NOTIFICATION_STATUS.READ } },
+      { $set: { read: NOTIFICATION_STATUS.READ } },
```

Comment:
I feel this is more readable as true or false
```suggestion
    { $set: { read: true } },
```